auto_cpufreq/core.py

```diff
@@ -676,8 +676,13 @@ def set_powersave():
         Path("/sys/devices/system/cpu/cpu0/cpufreq/energy_performance_preference").exists()
         and Path("/sys/devices/system/cpu/intel_pstate/hwp_dynamic_boost").exists() is False
     ):
-        run("cpufreqctl.auto-cpufreq --epp --set=balance_power", shell=True)
-        print('Setting to use: "balance_power" EPP')
+        if conf.has_option("battery", "energy_performance_preference"):
+            epp = conf["battery"]["energy_performance_preference"]
+            run(f"cpufreqctl.auto-cpufreq --epp --set={epp}", shell=True)
+            print(f'Setting to use: "{epp}" EPP')
+        else:
+            run("cpufreqctl.auto-cpufreq --epp --set=balance_power", shell=True)
```

Comment:
Do we now get in situation where if nothing was defined for `energy_performance_preference` in config file (or user isn't using config file), `balance_performance` will be used in both cases when computer is in charging & on power/battery? Or am I missing something?

Reply:
Never-mind I overlooked what was happened here.